src/context/whoami/WhoAmI.tsx

```diff
@@ -20,6 +20,12 @@ export class WhoAmI {
     return this.me.displayName;
   }
 
+  public getRealm(): string {
```

Comment:
There is no need to add a return type on this method as the type can be inferred from the returned value.
```suggestion
  public getRealm()
```